eclair-core/src/test/scala/fr/acinq/eclair/blockchain/electrum/ElectrumClientSpec.scala

```diff
@@ -69,15 +69,19 @@ class ElectrumClientSpec extends TestKit(ActorSystem("test")) with FunSuiteLike
 
   test("get transaction id from position") {
     probe.send(client, GetTransactionIdFromPosition(height, position))
-    val GetTransactionIdFromPositionResponse(txid, merkle) = probe.expectMsgType[GetTransactionIdFromPositionResponse]
+    val GetTransactionIdFromPositionResponse(txid, height1, position1, merkle) = probe.expectMsgType[GetTransactionIdFromPositionResponse]
     assert(txid === referenceTx.txid)
+    assert(height1 === height)
+    assert(position1 === position)
     assert(merkle === Nil)
   }
```

Comment:
nitpick: since we know the expected response, we could instead use `probe.expectMsg()` here